vtkext/private/module/vtkF3DRenderer.cxx

```diff
@@ -338,7 +336,7 @@ void vtkF3DRenderer::InitializeUpVector(const std::string& upString)
   }
   else
   {
-    F3DLog::Print(F3DLog::Severity::Warning, upString + " is not a valid up direction");
+    F3DLog::Print(F3DLog::Severity::Warning, upString + " is not a valid up direction\n");
```

Comment:
Don't we already add a `\n` automatically?

Reply:
Ill need to recheck on that.

Reply:
we do not, Ive fixed all instance, but I could put that in a dedicated MR.

Reply:
> I could put that in a dedicated MR.

Would be better yes